s3/pkg/service/multipart.go

```diff
@@ -238,16 +240,18 @@ func (s *s3Service) UploadPart(ctx context.Context, stream pb.S3_UploadPartStrea
 		return err
 	}
 
+	md5Writer := md5.New()
```

Comment:
IMO, as ETag is calculated here now, then each backend driver no need to calculate it again, but just get the ETag from the backend storage if it provided, so the backend driver also need to do some change, right?

Reply:
I think we should no longer check the ETag from any backend, so it is not necessary to return ETag from backend.

Reply:
But what if data damaged during transferring?

Reply:
so I will add the etag check in s3 grpc server layer, and also pass etag to backend ?